utils/check/handle_rpms.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -uex
+
+# This file does an install of the RPMs found in "${REPO_DIR}" and then
+# finds all the packages that the binaries installed reference.
+
+# The file ${PACKAGES} will contain a list of packages that the RPMs depend
+# on.
+
+# The file ${SOFILES} will contain a list of sharable images depended on by
+# the RPMs.  The default name for ${SOFILES} is "daos_depends_libries".
+
+# This will likely need to be edited for other build systems.
+
+: "${WORKSPACE:="${PWD}"}"
+
+: "${RPM_DIR:="${WORKSPACE}/rpm_dir"}"
+
+: "${SOFILES:="daos_depends_libraries"}"
+: "${PACKAGES:="daos_depends_packages"}"
+
+
+repo_file="/etc/yum.repos.d/"
+repo_file+="repo.dc.hpdd.intel.com_repository_daos-stack-"
+repo_file+="el-7-x86_64-stable-local.repo"
+
+repo_url="https://repo.dc.hpdd.intel.com/repository/"
+repo_url+="daos-stack-el-7-x86_64-stable-local"
+
+if [ ! -e "${repo_file}" ]; then
+  cat << EOF > daos_repo_file
+
+[repo.dc.hpdd.intel.com_repository_daos-stack-el-7-x86_64-stable-local]
+name=added from: ${repo_url}
+baseurl=${repo_url}
+enabled=1
+
+gpgcheck = False
+
+EOF
+  sudo cp daos_repo_file "${repo_file}"
+  sudo yum -y makecache
+fi
+
+sudo yum -y remove cart cart-debuginfo cart-devel cart-tests \
+                   daos daos-client daos-debuginfo daos-devel daos-server \
+                   daos-tests CUnit dpdk fio fuse fuse-lib hwloc \
+                   libabt0 libcmocka libfabric libfabric-devel libisa-l \
+                   libopa1 libpmem libpmemblk mercury ndctl\
+                   ompi openmpi3 openpa-devel \
+                   pexpect protobuf-c pmix \
+                   python-configshell python-urwid \
+                   raft raft-debuginfo raft-devel \
+                   spdk spdk-tools
+
+sudo yum -y install "${RPM_DIR}"/*.rpm
+
+set +x
```

Comment:
Should never do this.  Add the required error handling.

Reply:
I do not understand this comment.  The intention is to install all the non-source rpms from a specific Jenkins build with out needing to know what they are, and is not limited to just DAOS builds.

Reply:
set -x / set +x does not normally affect code flow.  Are you confusing it with +e?

The section of the script below that point is a loop, and with set -x on, it floods the console with effectively useless text.